apps/demos/Demos/Scheduler/Templates/Vue/MovieInfoForm.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <div class="movie-info-container">
+    <div
+      v-if="movie"
+      class="movie-info"
+    >
+      <div class="movie-preview-image">
+        <img
+          :src="movie.image"
+          :alt="movie.text"
+        >
+      </div>
+      <div class="movie-details">
+        <div class="title">{{ movie.text }} ({{ movie.year }})</div>
+        <div>Director: {{ movie.director }}</div>
+        <div>Duration: {{ movie.duration }} minutes</div>
+      </div>
+    </div>
+    <div
+      v-else
+      class="movie-info"
+    >
+      <div class="movie-preview-image"/>
+      <div class="movie-details">
+        <div class="title">Select a movie</div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import type { MovieResource } from './data';
+
+defineProps<{
+  movie: MovieResource | null | undefined;
+}>();
+</script>
+
+<style scoped>
+.movie-info-container {
```

Comment:
These styles are duplicate. We already have them in the Vue/styles.css. Let's remove them from here